Knossos.NET/Views/Windows/ModSettingsView.axaml

```diff
@@ -34,6 +34,17 @@
 						<Label Foreground="White">Available exec type options: </Label>
 						<Label Foreground="Orange">Default, Release, Debug, Fred2, Fred2Debug, QtFred, QtFredDebug</Label>
 					</WrapPanel>
+					<!--Mod Engine Settings-->
+					<Label Margin="0,20,0,0" FontWeight="Black" FontSize="20">ENGINE SETTINGS</Label>
+					<CheckBox IsChecked="{Binding NoIngameOptions}" Margin="5" ToolTip.Tip="By disabling ingame options, launcher engine settings will be used instead. Only valid if the mod uses a FSO engine build equal or higher than version 24.2.0">
```

Comment:
Nice, just small suggestion: to make consistent with other public facing uses of the word it might be best to use `in-game`. Also do we want this tooltip sentence and the others to end in a period. Right now one does but others do not. That's of course very minor but just something I noticed :D